content/en/security_platform/default_rules/_index.md

```diff
@@ -9,6 +9,10 @@ aliases:
 disable_sidebar: true
 ---
 
-Datadog provides out-of-the-box (OOTB) detection rules to flag attacker techniques and potential misconfigurations so that you can immediately take steps to remediate. Datadog continuously develops new default detection rules, which are automatically imported into your account.
+Datadog provides out-of-the-box (OOTB) detection rules to flag attacker techniques and potential misconfigurations so you can immediately take steps to remediate. Datadog continuously develops new default rules, which are automatically imported into your account, your Application Security library, and with the Agent, depending on your configuration. For more information, see the Detection Rules documentation.
```

Comment:
```suggestion
Datadog provides out-of-the-box (OOTB) detection rules to flag attacker techniques and potential misconfigurations so you can immediately take steps to remediate. Datadog continuously develops new default rules, which are automatically imported into your account, your Application Security library, and the Agent, depending on your configuration. For more information, see the Detection Rules documentation.
```

Reply:
Also, is there a link for "Detection Rules documentation"?

Reply:
Why yes, there is! ✨  Added.